reflection.md

```diff
@@ -63,4 +63,8 @@ Concatenated `Literal`s.
    Thus, two things I'd definitely want to implement are 
     1. some operator(s) that would allow representation of all characters/common 
     patterns/sets, and 
-    2. a negation operator
\ No newline at end of file
+    2. a negation operator
+
+   I feel that neither of these should be particularly challenging in Scala, and
+   should be a matter of adding more case classes in the IR component to handle
+   them.
```

Comment:
I think these are great ideas and would be very useful for a Regular Expressions DSL. I also think having an anything character would be as simple as copy/pasting the EPSILON line and putting ANYTHING instead.